webinar.html

```diff
@@ -0,0 +1,231 @@
+<li><strong>2020/10/23</strong>: <a href="https://wp.doc.ic.ac.uk/bglocker/talks/">  Domain Adaptation and Representation Transfer </a> </li>
+<li><strong>2020/10/16</strong>: <a href="https://www.youtube.com/watch?v=fmVuNRKsQa8&list=PLpqTCw5MzPU38pe8p6OfpVg8U14iot4mh&index=17&ab_channel=InstituteforPure%26AppliedMathematics%28IPAM%29">  Michael Mahoney: "Why Deep Learning Works: Implicit Self-Regularization in Deep Neural Networks"  </a> </li>
+<li><strong>2020/10/9</strong>: <a href="https://www.youtube.com/watch?v=BwAVyB6z4ok&list=WL&index=70&ab_channel=InstituteforAdvancedStudy">  Are All Features Created Equal? - Aleksander Madry  </a> </li>
+<li><strong>2020/9/25</strong>: <a href="https://www.youtube.com/watch?v=98KiH44yvok&t=1238s&ab_channel=InstituteforPure%26AppliedMathematics%28IPAM%29"> Stéphane Mallat: "Deep Generative Networks as Inverse Problems"   </a> </li>
+<li><strong>2020/9/18</strong>: <a href="https://www.youtube.com/watch?v=USfNJNePDKQ&t=539s">   Representational Power of Graph Neural Networks - Stefanie Jegelka  </a> </li>
+<li><strong>2020/9/11</strong>: <a href="https://www.youtube.com/watch?v=IXotICNx2qk&list=PLpqTCw5MzPU38pe8p6OfpVg8U14iot4mh&index=20&t=0s&ab_channel=InstituteforAdvancedStudy">  Is Optimization the Right Language to Understand Deep Learning? - Sanjeev Arora  </a> </li>
+<li><strong>2020/9/4</strong>: <a href="https://www.youtube.com/watch?v=S27pHKBEp30&ab_channel=SeattleAppliedDeepLearning"> LSTM is dead. Long Live Transformers!   </a> </li>
+
+<h2>
+COVID Break
+</h2>
+
+<li><strong>2019/10/03</strong>: <a href="https://www.youtube.com/watch?v=8CenT_4HWyY">3D deep learning (CVPR 2017)</a> : 43:00 to 1:08:00</li>
+<li><strong>2019/09/19</strong>:&nbsp;</strong><a href="https://www.youtube.com/watch?v=Bo8MY4JpiXE">Interview with Francois Cholet.</a></li>
+<li><strong>2019/09/12</strong>:&nbsp;<a href="https://www.youtube.com/watch?v=_-RU9Yoca84">Capsule Networks for computer vision â CVPR 2019</a></li>
+<li><strong>2019/09/05</strong>: <a href="https://www.youtube.com/playlist?list=PLe5rNUydzV9QHe8VDStpU0o8Yp63OecdW">Introduction on deep Bayesian methods</a></li>
+<li><strong>2019/08/01:</strong>&nbsp;<a href="http://A new perspective on adversarial-perturbations">A new perspective on adversarial-perturbations</a></li>
+<li><strong>2019/07/11-18:</strong>&nbsp;<a href="https://www.youtube.com/watch?v=KDRN-FyyqK0">ICML 2018: Tutorial Session: Toward the Theoretical Understanding of Deep Learning</a></li>
+<li><strong>2019/07/04</strong>: <a href="https://www.youtube.com/watch?v=z1xs9jdZnuY">Attention is all you need</a></li>
+<li><strong>2019/06/27:</strong>&nbsp;<a href="https://cbmm.mit.edu/video/principles-and-applications-relational-inductive-biases-deep-learning">Principles and applications of relational inductive biases in deep learning</a>&nbsp;by&nbsp;Kelsey Allen</li>
```

Comment:
This one was itself an embedded youtube video, so to embed it would go like this:
```suggestion
<li><strong>2019/06/27:</strong>&nbsp;Principles and applications of relational inductive biases in deep learning&nbsp;by&nbsp;Kelsey Allen</li>
<iframe width="560" height="315" src="https://www.youtube.com/embed/sTGKOUzIpaQ" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```